.cspell.json

```
@@ -1525,6 +1525,7 @@
     "tfar",
     "TFLOPS",
     "theora",
+    "Thinkpad",
```

The official notation is `ThinkPad`, so it's good to add the lowercase word, considering the usage when written in URLs (as your reference link).

```suggestion
    "thinkpad",
```

---

@Yukio247 What do you think?

---

@kenji-miyake 
Thank you. It would be fine with me!